docs/4.0/components/forms.md

```diff
@@ -1162,8 +1162,17 @@ The [`:lang()` pseudo-class](https://developer.mozilla.org/en-US/docs/Web/CSS/:l
 {% highlight scss %}
 $custom-file-text: (
   en: "Browse",
-  es: "Navegar"
+  es: "Elegir"
```

Comment:
“Browse” is usually translated as _Examinar_ (see, e.g., Microsoft Terminology)

Reply:
Please open a new issue :).